cobrawap/pipeline/stage03_trigger_detection/scripts/calc_thresholds_fitted.py

```diff
@@ -153,4 +153,4 @@ def fit_amplitude_distribution(signal, sigma_factor, fit_function,
                                        args.img_name.replace('_channel0', f'_channel{channel}'))
             save_plot(output_path)
 
-    np.save(args.output, thresholds)
\ No newline at end of file
+    np.save(args.output, thresholds)
```

Comment:
Files should end with a newline.

Reply:
Same as above, it ends with a blank line.